target/linux/mpc85xx/files/arch/powerpc/boot/dts/red-50-rev1.dts

```diff
@@ -0,0 +1,263 @@
+// SPDX-License-Identifier: GPL-2.0-or-later or MIT
```

Comment:
Any plans to upstreaming this DTS to Linux kernel?

Reply:
You should give it a try, and you will see. Don't be afraid of new adventures or challenges in your life. :)

Reply:
Before that, I need another device to make sure that USB port is powered. Currently I cannot risk to brick this device. Hopefully I'll get one the next weeks.

If I upstream it, it should be really perfekt.